Makefile

```diff
@@ -18,8 +18,9 @@ TESTS = $(shell find test -name "*.js" -type f | sort)
 
 all: mocha.js
 
-mocha.js: $(SRC) browser-entry.js
+mocha.js BUILDTMP/mocha.js: $(SRC) browser-entry.js
```

Comment:
On a successful test run the directory is cleaned out. This happens in the npm test script. Changing the directory name is a minor thing. I'm open to suggestions of something better